advocacy_docs/partner_docs/QuestFoglightforDatabases/04-ConfiguringQuestFoglightforDatabases.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Configuring Quest Foglight for Databases'
+description: 'Walkthrough on configuring Quest Foglight for Databases'
+---
+
+Implementing Quest Foglight for Databases with EDB Postgres Advanced Server and/or EDB Postgres Extended Server requires the following components:
+
+- EDB Postgres Advanced Server and/or EDB Postgres Extended Server
+- Quest Foglight Software
+
+### Prerequisites
+
+- A running EDB Postgres Advanced Server or EDB Postgres Extended Server instance
+- Quest Foglight Software installed
+- Login credentials for your EDB Advanced Server or EDB Postgres Extended instance and Quest Foglight interface
```

Comment:
Should be "... EDB Postgres Advanced Server or EDB Postgres Extended Server instance"